File: python_programming_basics/conditional_statements_advanced/e_07_hotel_room.py
month = input()
sleeps_count = int(input())
studio_price = 0
apartment_price = 0

if month == "May" or month == "October":
    studio_price = sleeps_count * 50
    apartment_price = sleeps_count * 65

    if sleeps_count > 14:
        studio_price -= studio_price * 0.3

    elif sleeps_count > 7:
        studio_price -= studio_price * 0.05

elif month == "June" or month == "September":
    studio_price = sleeps_count * 75.2
    apartment_price = sleeps_count * 68.7

    if sleeps_count > 14:
        studio_price -= studio_price * 0.2

elif month == "July" or month == "August":
    studio_price = sleeps_count * 76
    apartment_price = sleeps_count * 77

if sleeps_count > 14:
    apartment_price -= apartment_price * 0.1

print(f"Apartment: {apartment_price:.2f} lv.")
print(f"Studio: {studio_price:.2f} lv.")

File: JavaScript Basics/03. Conditional Statements Advanced/e-07-Hotel-Room.js
function calculateHotelCost(month, nights) {
    let studioPricePerNight;
    let apartmentPricePerNight;

    if (month === "May" || month === "October") {
        studioPricePerNight = 50;
        apartmentPricePerNight = 65;
    } else if (month === "June" || month === "September") {
        studioPricePerNight = 75.20;
        apartmentPricePerNight = 68.70;
    } else if (month === "July" || month === "August") {
        studioPricePerNight = 76;
        apartmentPricePerNight = 77;
    }

    let studioCost = studioPricePerNight * nights;
    let apartmentCost = apartmentPricePerNight * nights;

    if ((month === "May" || month === "October") && nights > 14) {
        studioCost *= 0.7;
    } else if ((month === "May" || month === "October") && nights > 7) {
        studioCost *= 0.95;
    } else if ((month === "June" || month === "September") && nights > 14) {
        studioCost *= 0.8;
    }


    if (nights > 14) {
        apartmentCost *= 0.9;
    }

    studioCost = studioCost.toFixed(2);
    apartmentCost = apartmentCost.toFixed(2);

    console.log(`Apartment: ${apartmentCost} lv.`);
    console.log(`Studio: ${studioCost} lv.`);
}


calculateHotelCost("May", 15);
calculateHotelCost("June", 14);
calculateHotelCost("August", 20);
